### PR TITLE
FT2-893 Review custom event not being logged. Added code for the 'Rev…

### PR DESCRIPTION
**Pull request by Gary-Griffiths-Methods**
…iew' audit trail event.

### CODE REVIEW

File: DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs [hunk 1]
```diff
@@ -105,19 +106,31 @@ public override Task CreateAsync(AuditTrailCreateContext context)
                                         break;
 
                                     case "submit.RequestApproval-ContentDesign":
-                                        contentEvent.Name = "Requested for review (ContentDesign)";
+                                        reviewType = "content design";
+                                        contentEvent.Name = contentApprovalPart.ReviewStatus != Models.Enums.ReviewStatus.InReview
```
Comment (ravi-methods) [inline DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs:110]: Could we extract these 3 lines to a local method and call it SetContentEventName or something? This is being used in 4 places.

Comment (ravi-methods) [reply DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs:110]: Or rather GetContentEventName, so you do the assignment here by just getting the value from that method.

Comment (Gary-Griffiths-Methods) [reply DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs]: Added GetContentEventName() method

File: DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs [hunk 2]
```diff
@@ -136,5 +136,13 @@ public override Task CreateAsync(AuditTrailCreateContext context)
             return Task.CompletedTask;
         }
 
+        private static string GetContentEventName(AuditTrailCreateContext<AuditTrailContentEvent> contentEvent, ContentApprovalPart contentApprovalPart, string reviewType)
+        {
+            contentEvent.Name = contentApprovalPart.ReviewStatus != Models.Enums.ReviewStatus.InReview
```
Comment (ravi-methods) [inline DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs:141]: Don't need to pass contentEvent, just return the string value itself without assigning it to anything.

File: DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs (hunk 1 above)
Comment (Gary-Griffiths-Methods) [reply DFC.ServiceTaxonomy.ContentApproval/Handlers/ContentApprovalAuditTrailEventHandler.cs]: Removed ContentEvent parameter.

**Review by ravi-methods — APPROVED** [review]

**Pull request merged**